sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 5.0.2 (Unreleased)
-
+## 5.1.0b1 (Unreleased)
```

Comment:
do we want to update release date now or in a separate PR

Reply:
Changed to 2020-04-06